.codex/START_HERE.md

```
@@ -0,0 +1,38 @@
+# uc Start Here
+
+## 5-Minute Bootstrap
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)